Bad peer dep version of ethers

Hey CoW Protocol team!

Attempting to add `@cowprotocol/contracts` as a dep in our repo makes npm throw an error. We're using ethers v5.6.0 while this repo seems to be using [ethers v5.5.2](https://github.com/cowprotocol/contracts/blob/9bbf72e3c0f9cc71f3da98f3e0fe3383d0a7abba/package.json#L65) while [specifying `~5.4.0` as a peer dep](https://github.com/cowprotocol/contracts/blob/9bbf72e3c0f9cc71f3da98f3e0fe3383d0a7abba/package.json#L33). This makes npm v7 throw an error (see below). 

Since you already have ethers as a dep, can you just remove the peer dependency altogether? Is there a better solution?

```
$ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: nexusmutual@1.5.0
npm ERR! Found: ethers@5.6.0
npm ERR! node_modules/ethers
npm ERR!   ethers@"5.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer ethers@"~5.4.0" from @cowprotocol/contracts@1.3.0
npm ERR! node_modules/@cowprotocol/contracts
npm ERR!   dev @cowprotocol/contracts@"^1.3.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/user/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2022-06-14T10_15_20_321Z-debug-0.log
```

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
     "prepack": "yarn build"
   },
   "peerDependencies": {
-    "ethers": "~5.4.0"
+    "ethers": "^5.4.0"
   },
   "devDependencies": {
     "@0x/contract-artifacts-v2": "npm:@0x/contract-artifacts@^2.2.2",

